Alexey-budgetCalc/modul.py

```diff
@@ -0,0 +1,211 @@
+import csv
+import matplotlib.pyplot as plt
+
+
+def month_input():
+    month = ""
+    flag = True
+    while flag:
+        month = input("Enter the month in format YYYY-MM \n")
+        if (len(month) == 7) and (month[4] == "-"):
```

Comment:
Check regular expressions (`re` library). Such concept is widely used in all projects to match strings by pattern